sklearn/feature_selection/rfe.py

```diff
@@ -312,6 +312,10 @@ class RFECV(RFE, MetaEstimatorMixin):
         Refer :ref:`User Guide <cross_validation>` for the various
         cross-validation strategies that can be used here.
 
+        .. deprecated:: 0.20
```

Comment:
Can you make this versionchanged instead of deprecated? (because the keyword itself is not deprecated)

Reply:
done